src/AstroBase.jl

```diff
@@ -1,5 +1,9 @@
 module AstroBase
 
-# package code goes here
+using Rotations
+export pom00
+@inline function pom00(rx, ry, sp)
```

Comment:
The `@inline` here is unnecessay. A very short function like this will be inlined automatically. `@inline` should generally be only applied after careful benchmarking...